### ISSUE TITLE
Rendering Latency

### ISSUE THREAD

**Issue by cdemirsoy** [unrealcv/unrealcv#116]
Hi,

I have a very important question, did anyone use unrealcv to create training data for deep learning when there are moving objects? I think at this point this is not possible because different and nondeterministic rendering time and  averts to have exactly same frame both masked and non-masked.

I am thinking on it for days but I had no progress. Please free feel to share your opinion!

Best Regards,
Canberk

**Comment by qiuwch** [comment]
I am using it for this purpose, but what I am doing now requires some knowledge of using blueprint in UE4. The idea is calling unrealcv functions in the blueprint and save the image and annotation in the same tick of the game.

Another approach is pausing the animation during the data capture and resume the animation after that.

**Comment by martijn-vinotion** [comment]
Hi,

I am also using unrealcv for this purpose. I am trying to use commands to pause and unpause the game but I found that UnrealCV currently only has the pause functionality implemented while the resume command is commented out in code (and causes errors when uncommented and trying to compile).

Does anyone have a solution for this?
I see that UE4 has [this method](https://api.unrealengine.com/INT/API/Runtime/Engine/GameFramework/APlayerController/SetPause/index.html) that can probably be used to unpause the game.

Thanks in advance!

**Comment by martijn-vinotion** [comment]
I implemented the resume action in [this pull request](https://github.com/unrealcv/unrealcv/pull/139).

**Comment by cdemirsoy** [comment]
You can directly use UE4 commands by using UnrealCV.

**Comment by isalirezag** [comment]
@cdemirsoy @martijn-vinotion can tell us how did you solve it?
@qiuwch can you tell how to do it in blueprint?

**Comment by cdemirsoy** [comment]
I finalized this project of mine 14 months ago. Solution was basically running a frame, pausing, taking a screenshot, applying binary filter to human body, taking one more screenshot and then going to next frame and repeating it.

So eventually I had a data set with ground truth.

I could pause the simulation via Python API and that's it.

**Comment by isalirezag** [comment]
@cdemirsoy I see, can you please tell me how did you pause? did you use `vset /action/keyboard` ? if so do you know what `delta` should be in this command??
